### ISSUE TITLE
Product Elements: Adding “text styling toolbar” to Add to cart button

### ISSUE THREAD

**Issue by imanish003** [woothemes/woocommerce#42425]
We wanna add following toolbar controls to Add to cart button toolbar, which is also available on core/button.
![image](https://user-images.githubusercontent.com/16707866/227856350-2fd07eb1-d847-4a7f-8168-9560b40aeea5.png)

For more info, check following GitHub discussion: https://github.com/woocommerce/woocommerce-blocks/discussions/8848

This was part of https://github.com/woocommerce/woocommerce-blocks/issues/8085, but I am creating this new issue to unblock https://github.com/woocommerce/woocommerce-blocks/issues/8085.

**Comment by imanish003** [comment]
As discussed on #8848, I am descoping this task for now. 
We agreed that the available typography options from the sidebar are sufficient for the button and that adding the "text styling toolbar" is not necessary unless there is sufficient demand from community or direction from Shaun to implement it in the future.
